hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -556,9 +556,22 @@ public Iterator<? extends OzoneKey> listKeys(String keyPrefix)
    */
   public Iterator<? extends OzoneKey> listKeys(String keyPrefix, String prevKey)
       throws IOException {
+    return listKeys(keyPrefix, prevKey, false);
+  }
 
+  /**
+   * Returns Iterator to iterate over all keys after prevKey in the bucket.
+   * If fastList is true, iterator will only contain immediate children.
+   *
+   * @param keyPrefix Bucket prefix to match
+   * @param prevKey Keys will be listed after this key name
+   * @param fastList Only list immediate children ozoneKeys if fastList is true
```

Comment:
Would be nice if we could find a more descriptive name for the new param.  `fastList` begs the question: why would anyone want a slow list? :)

Reply:
> Would be nice if we could find a more descriptive name for the new param. `fastList` begs the question: why would anyone want a slow list? :)

Just change `fastList` -> `simpleList`. (Will follow up If there is a better param)

Reply:
How about `shallow`?

Reply:
> How about `shallow`?

Ok, it's fine to me . I can refactor it together after the code review if any other change

Reply:
> I can refactor it together after the code review if any other change

Makes sense, thanks.